### DISCUSSION TITLE
[Enhancement] Add the icon picker on zone edit dialog

### DISCUSSION THREAD

**Discussion by theyapps** [home-assistant/frontend#10553] [Other feature requests] [answered]
With the addition of the [icon picker](https://www.home-assistant.io/blog/2021/11/03/release-202111/#the-icon-picker) in the 2021.11 release.

I have noticed that the zone editor dialog does not have this functionality enabled.
![image](https://user-images.githubusercontent.com/15441/140597484-145cba9e-eaf7-44ba-a972-2fe32515556b.png)

**Comment by theyapps** [comment, marked as answer]
I have implemented this functionality locally and will work on committing and doing a PR 

![image](https://user-images.githubusercontent.com/15441/140599642-6b37e86d-80fd-4ca6-ae3b-35a6d00f7ac4.png)

**Reply by spacegaier** [reply]
The original PR has been superseded by https://github.com/home-assistant/frontend/pull/10447
